index.html

```diff
@@ -121,8 +121,12 @@
             firstDiv.removeChild(firstDiv.firstChild);
           }
           if (addImage < 7 && !alreadyHaveImage) {
+            //randomly pick an image from data.txt
+            var imgNum = getRandomInt(0, imgArray.length - 1);
+            console.log(imgArray.length);
+            console.log(imgNum);
```

Comment:
Please take out the `console.log()` bits. Code that's used during development should generally be removed before getting it out to production. (One of the things I've been trained to look at when doing reviews : )